.github/workflows/continuous-integration.yaml

```diff
@@ -728,8 +731,6 @@ jobs:
             fi
           fi
 
-          echo "FAILED_TESTS_ARE_FLAKY=${FAILED_TESTS_ARE_FLAKY}" >> "$GITHUB_ENV"
```

Comment:
basically the problem is that this step needs `shell: bash +e {0}`, which made this echo cancel the above `exit 1` command in case of test or coverage failure !